src/resources/ResourceSnapshots/ResourceSnapshotsInterfaces.ts

```diff
@@ -1,12 +1,39 @@
 export interface ResourceSnapshotsModel {
+    /**
+     * The username of the snapshot creator.
+     */
     createdBy: string;
+    /**
+     * The date of creation of the snapshot in number of milliseconds since UNIX epoch.
+     */
     createdDate: number;
+    /**
+     * A note you can attach to your snapshot.
+     */
     developerNote?: string;
+    /**
+     * The unique identifier of the target snapshot.
+     */
     id: string;
+    /**
+     * The unique identifier of the organization from whichthe snapshot originates, if it differs from the current organization.
```

Comment:
```suggestion
     * The unique identifier of the organization from which the snapshot originates if it differs from the current organization.
```

Reply:
Good point. On my end, I just copied the doc from https://docs.coveo.com/en/3432/cloud-v2-api-reference/migration-api#operation/createSnapshotUsingPOST_2.
So it also has to be fixed out there.